src/styles/index.scss

```diff
@@ -1,3 +1,134 @@
-body {
+@import './variables';
+
+.page-body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-size: 160px 134px;
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    width: 100%;
+    color: $main-color;
+    font-size: 12px;
+    font-weight: 500;
+    font-family: inherit;
```

Comment:
The `font-family` property is inherited by default, so there's no need to declare it again here. You've already declared it in the `.page-body` class, so it will be inherited by all children elements.